tests/fixtures/callback/dir_store/foreman/testhost-facts.json

```diff
@@ -0,0 +1,12 @@
+{
+  "facts": {
+    "_timestamp": "2000-01-01 12:00:00+00:00",
+    "_type": "ansible",
+    "ansible_facts": {
+      "crypt": "ENCRYPTED_VAULT_VALUE_NOT_REPORTED",
+      "geheim": "admin",
```

Comment:
yes, the unencrypted version is stored, as that is not marked as "nolog" or anything :/